### ISSUE TITLE
Chapter *(and page) numbers starting at 0?

### ISSUE THREAD

**Issue by BaronKrause** [Daiz/manga-naming-scheme#3]
Just curious how manga with chapters that start at 0 (whether it be a prologue or just the first normal chapter) should be handled. Should we follow each mangas original system, having some starting at c000 and some starting with c001 (especially since the chapter numbers may actually listed inside the manga itself) or should we change everything to start at either c000 or c001?

Same goes for page numbers, those are usually all over the place in various downloaded rips. Some Scanlators choosing to start at 0, some start at 1, some have things like pages listed 01-25 with a page labeled 001 to ensure its at the start. Some magazine scans start at the page in the magazine they physically came from (like 116-132) with pages they added labeled differently (like a page just called "Credits" at the end). I'm assuming we should ignore all that and go with a standard numbering order (even on scans that have page numbers on there) since this is simply to denote the numerical order of the pages in this file, correct? If so should it start at p000 or p001?

Also, if a manga title has a character that is simply unsupported in a windows file name (like a "?" or ":"), should it just be omitted, or should something else go there? Chrome will for instance when saving a web page replace unsupported characters like "?" with a "_"). I'm assuming it should just be omitted, making a title like "What:The" or "WhatThe?" become just "WhatThe",  but figured I would ask.

Thanks!

**Comment by Daiz** [comment]
Chapter numbering should generally always follow the original numbering, so if a series starts at a numbered chapter 0, then it should be labeled c000. The exception would be if you have an extra chapter eg. at the end of volume one (which is say, 5 first chapters) labeled as "chapter 0", then you'd do c005x1 (since that's where it's positioned in the original order) and possibly use extra information to give it a descriptive label of [Chapter 0] or such.

For pages, you should always start at p001, regardless of original numbering.

For unsupported characters in filenames, I recommend just stripping them out (or replacing with spaces depending on what you're exactly replacing), so "What:The?" could become either "WhatThe" or "What The".

**Comment by BaronKrause** [comment]
Thanks!

How would you handle a series that has no clear chapter numbering (Just chapter names in the initial mag release and in the later released tank) but with an extra epilogue like mini chapter at the end on the tank release. Should it be lets say 008x1 with the [Omake] extra info tag like you would if they had clear chapter numbers (like c008x1) or should the extra omake just be 009 with no extra tag in this case?

**Comment by Daiz** [comment]
Since tankoubon releases inevitably come after the magazine release, 008 and 009 would usually exist already, thus the extra chapter should be labeled 008x1.

**Comment by BaronKrause** [comment]
Cool sounds good!

OK, last questions I swear! All my odd ball sets that didn't fit into the guidelines cleanly should be covered after.

I have an extra chapter where chronologically it was released in between 17 and 18 in the 3-4 month gap between chapters in a special anniversary edition of that magazine. It has no actual numbering. Manga sites have it listed anywhere from chapter 20.5-23.5 (months to a year after its release) and the scanlators think they remember it going between 19-20 (still a few months after its release, but it could've been planned). There is an official 17.5 released in the following month's issue.
Gut is to place this chapter after 17, and labeled as 17x1 and making the officially labeled 17.5 that came out the following month 17x2.

Or should it not be given a number at all and just called 000 since its honestly just an educated guess.

Should the same be done for nonsense parody mini chapters that might of came out separately or been released as a promotion for something, that have absolutely no canon placement in a story? Offhand I know one of my series has a BS Haloween parody non chapter that really is nothing but an art exhibition along with a LINE sticker advertisement.

Thanks for your work with this!

**Comment by Daiz** [comment]
>Gut is to place this chapter after 17, and labeled as 17x1 and making the officially labeled 17.5 that came out the following month 17x2.

Yes, this is what I would recommend doing. Also add an extra information tag for both, maybe something like [Magazine Anniversary Extra] or such, while the other could be just [Chapter 17.5] or so.

For parody mini chapters, if they're completely irrelevant and there's many of them they could be released under a different title completely, eg. if you Main Series then it could be Main Series Promotional Shorts and then just labeled with chronological chapter numbers there.

Otherwise I'd just recommend sticking them as extras in chronological order, ie. place them after whichever chapter they came out, label with extra info as eg. [Halloween Line Sticker Promo Short] or such.

**Comment by BaronKrause** [comment]
Thanks!

If you don't mind 1 last question, what is your criteria for determining if the English title is from the original author? A manga like "Ninja Shinobu-san no Junjou" has an English title under the Japanese one on ever cover ("Shinobu's Pure Heart") so it is pretty obvious there that the original author wanted it to be read that way in English.

Others like "Kaguya-sama wa Kokurasetai" (it's actually "Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen", I dont imagine adding anything past the ":" should be added to the file name though) that has no official English name mentioned in the text has just been licenced and is called "Kaguya-sama: Love is War". There is no mention of that name before, do you count English names that poped up after licencing as from the original author? Im guessing no (as chances are that name diddnt even come from the author) and would cause a PITA issue of redoing previously converted chapters from pre licencing.

**Comment by Daiz** [comment]
As the scheme states:

>New releases **must** use the same title as older releases.

In other words, the first release basically decides it all. Since these will usually be done from a Japanese release, whether it be a magazine or tankoubon, if there's no English title to be found there then a romanized Japanese title is to be used, and all releases should follow that, even if an English title emerges later (whether from the original author or a licensed version).

Of course, if a first release only happens after say, an English licensed version has appeared, technically they could use that for the filenames, but I'd still recommend going with whatever the original had.

Though as an additional recommendation, if there is an English title in the original work but it's a hopelessly Engrish-y mess (consider eg. a LN-like long title that had the equivalent of a Google translated mess of an Engrish next to the original title), it'd probably be a better idea to go with a romanization of the JP name instead.

**Comment by Footprintxabc** [comment]
I got a questions related to digital manga volumes. Sometimes they come with extra pages before the very first chapter, such as the chapter index, author notes and etc. How are those numbered? The original page numbers for those are not actually numbers, but roman numerals. Would I use something like p000a, p000b, p000c... instead? Or should it start with p000 first? So p000, p000a, p000b, p000c? I'm assuming p000 would be the volume cover. This is just a guess for a possible syntax guideline. However, what if becomes way too lengthy? There are 6 pages for the volume I have before the first chapter. which would go up to p000f. Not sure if the would look kind of goofy. Hypothetically, what if it did not have page numbers? Meaning those pages did not have the roman numerals? What then?

However, going back to one of the replies your gave, you said "for pages, you should always start at p001, regardless of original numbering." So if I include the index and author notes and publishing notes, would they start from p001? Then the first page of the first chapter would be like p007? I feel like following original numbering can help others determine which pages is corrupted and can be replaced easier instead of having to add or subtract a certain number to make sense of it all. In this example, my page 1 would be page 7 in the file name. So if I go back to replace a corrected or corrupted page that the publisher fixed, I don't have to do mental calculations. It would be a hassle if there were more than 10 pages that are not consecutive. Comparing apple with apples simplify things. This is just my opinion, but what would I know. Just a friendly suggestion. I think your example might have been referring to magazines, like WSJ, where chapters can start at page 1, which is totally fine. But I feel like there isn't a rule to follow for volumes, especially official volume releases of manga. If a script can be created to do this tedious work, that would be so awesome. Just have to input what the situation is and state which page is the start of each chapter, so it can include it in the name.

If you could come up with a standard for groups to solve this out there, that would be awesome. Could you also update whatever was said under issues on the main page after every discussion? Much appreciated.
